### DISCUSSION TITLE
This app seems find a NEW way to detect lsposed

### DISCUSSION THREAD

**Discussion by diedbug2002** [LSPosed/LSPosed#2788] [General]
https://play.google.com/store/apps/details?id=com.sinopac.dawho&hl=zh_TW&gl=US

Recent update of this app seems able to detect whether lsposed is loaded or not.

Tested with both Magisk and KernelSU on Android 10, 12 with 0 lsposed module + shamiko enabled

Once Lsposed load, it crashed

Once Lsposed disable, it work again

Screenshot:
![18311_0](https://github.com/LSPosed/LSPosed/assets/25360103/1b47df65-3045-43a1-abdb-1ca8dbc14197)
![18310_0](https://github.com/LSPosed/LSPosed/assets/25360103/8f67594d-6d4e-4669-84c4-1b2c5b576b22)

Lsposed Log (LSPosed-v1.9.1-7006-zygisk):
[LSPosed_2023-10-05T19_46_55.315.zip](https://github.com/LSPosed/LSPosed/files/12819072/LSPosed_2023-10-05T19_46_55.315.zip)

Magisk Log (app added to blacklist):
[magisk_log_2023-10-05T19.48.02.log](https://github.com/LSPosed/LSPosed/files/12819119/magisk_log_2023-10-05T19.48.02.log)

**Comment by quyenvsp** [comment]
https://play.google.com/store/apps/details?id=com.vnid

same with this app from version 2.0.8
